app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -1,35 +1,46 @@
 <resources>
     <string name="app_name">Blue Line Console</string>
-    <string name="loading_main_caption">正在刷新数据库......</string>
+	
+    <string name="loading_main_caption">正在刷新数据库...</string>
```

Comment:
Sorry for unclear change. Can you change to &#8230; (same as English and Japanese) or keep original six dots?

English and Japanese text had three dots, and Linter claimed that three dots should be replaced with &#8230; . And checking the reason, it seemed better to follow the Linter for this. Six dots has no better alternative, so I did nothing to six dots.
https://stackoverflow.com/questions/8536375/android-replace-with-ellipsis-character